### ISSUE TITLE
OpenOCD fails to connect to Pico using Debug Probe on Windows 11

### ISSUE THREAD

**Issue by nscooling** [raspberrypi/pico-feedback#319]
When trying to connect to a Pico via the Raspberry Pi Debug Probe using OpenOCD, I get the following message:
```
❯ openocd -f interface/cmsis-dap.cfg -c "adapter speed 5000" -f target/rp2040.cfg -c "program build/test.elf verify reset exit"
Open On-Chip Debugger 0.11.0-ms1 (2022-02-14-22:06) [https://github.com/microsoft/openocd]
Licensed under GNU GPL v2
For bug reports, read
        http://openocd.org/doc/doxygen/bugs.html
adapter speed: 5000 kHz

Info : auto-selecting first available session transport "swd". To override use 'transport select <transport>'.
Info : Hardware thread awareness created
Info : Hardware thread awareness created
Info : RP2040 Flash Bank Command
Info : Using CMSIS-DAPv2 interface with VID:PID=0x2e8a:0x000c, serial=E6616407E3620D29
Info : CMSIS-DAP: SWD  Supported
Info : CMSIS-DAP: FW Version = 2.0.0
Info : CMSIS-DAP: Interface Initialised (SWD)
Info : SWCLK/TCK = 0 SWDIO/TMS = 0 TDI = 0 TDO = 0 nTRST = 0 nRESET = 0
Info : CMSIS-DAP: Interface ready
Info : clock speed 5000 kHz
Error: Sequence 4 not supported
Info : DAP init failed
in procedure 'program'
** OpenOCD init failed **
shutdown command invoked

Error: Sequence 3 not supported
Error: Sequence 4 not supported
```

The same setup works fine from my Mac.
```
❯ openocd --version
Open On-Chip Debugger 0.11.0-ms1 (2022-02-14-22:06) [https://github.com/microsoft/openocd]
```

**Comment by aallan** [comment]
How did you install OpenOCD, did you use the [new Windows Installer](https://www.raspberrypi.com/news/raspberry-pi-pico-windows-installer/) or some other method? In either case, this is probably the sort of question you should be asking on the [forums](https://forums.raspberrypi.com/viewforum.php?f=143) as you're going to get more feedback there. I'm closing the issue, but please come back and feedback if you identify a bug or a problem with the documentation.

**Comment by whaleygeek** [comment]
Another friend of mine that I was helping last week, independently reported that he didn't see the "Error sequence number not supported" messages, but that the end result was the same, in that his Pico Probe would not DAP Init. 

When I worked through this with him face to face, we found the probe worked on one pico dev target board, and didn't work on another. When he got home he reported that neither were working again. We wondered if there were marginal timing errors somewhere in the system (I have seen marginal timing errors before in my career where using a heat gun or freezer spray on the board has changed the behaviour). Note no heat gun or freezer spray to hand here to try that test!
